spec.md

```diff
@@ -292,6 +293,11 @@ use these error codes for the specified failure scenarios.
 | ConcurrencyError | The Service Broker does not support concurrent requests that mutate the same resource. | Clients MUST wait until pending requests have completed for the specified resources. |
 | RequiresApp | The request body is missing the `app_guid` field. | The `app_guid` MUST be included in the request body. |
 
+### Content Type
+
+All requests and responses defined in this specification with accompanying bodies MUST contain a `Content-Type` header set to `application/json`.
+If the `Content-Type` is not set, Service Brokers MUST respond with `400 Bad Request`.
```

Comment:
From my understanding, many `http server` implementations would add `Content-Type` header and set its default value to `application/json`, so I'm not quite sure if this constraint is required.

Reply:
For backwards compatibility, we might want to change both MUSTs here to SHOULD.

Reply:
@leonwanghui I don't think we should rely upon the potential behaviour of clients (the go http client for example does not set this header) or servers.

Seems like it's either part of the spec or not, and if a bunch of clients/servers fulfil that with no effort from the user that's great.

Reply:
@mattmcneeney I'm fine with either, I just think it should be specified if it's desirable behaviour (I think it is) because I futzed for an hour around writing a client for the `overview-broker` before I guessed this was what it was looking for. It's codified in all the curl snippets, but not mentioned anywhere else.